app/assets/javascripts/fae/form/_ajax.js

```diff
@@ -121,7 +121,21 @@ Fae.form.ajax = {
 
         // remotipart returns data inside textarea, let's grab it from there
         if ($html.is('textarea')) {
-          $html = $( $html.val() );
+          if ( $html.length > 1 ) {
+            var $textarea;
+
+            for (var i = 0; i < $html.length; i++ ) {
+              if ( $($html[i]).is('textarea') ) {
```

Comment:
Line 123 is not the same as ( $html === 'textarea'), is() responds true if any item in the array is a textarea.